lib/activities/email_link_confirm_activity.dart

```diff
@@ -1,8 +1,8 @@
-import 'package:flutter/material.dart';
-import 'package:corbado_demo/api/corbado_service.dart';
 import 'package:corbado_demo/activities/login_activity.dart';
```

Comment:
Hmm if we want to use this as an example for the passkeys package we should remove the email confirmation I think. This is something specific to the Corbado relying party server (so we should move it to an example for the corabdo_auth package).

Reply:
If we remove it there are other files affected as well (e.g. ContentActivity)

Reply:
Talked with Vincent, the repo should be an example for the corbado auth package - email confirmation will be integrated into corbado_auth as well, until then we have to use our custom impl for it